.travis.yml

```diff
@@ -45,6 +45,6 @@ jobs:
         docker exec -it "bats-dest-tests" bash -c 'bats -r -f "destructive-." .'
 # for now spellcheck tests are not enforced, return true to consider result as ok
     - stage: Shellcheck
-      script: shellcheck -x -s bash openhabian-setup.sh || true
-    - script: shellcheck -x -s bash functions/*.bash || true
-    - script: shellcheck -x -s bash build-image/*.bash || true
+      script: shellcheck -x -s bash openhabian-setup.sh
```

Comment:
since shellcheck is quiet if noting is found, you do not see any sign of it in the logs... I would rather write 
`script: shellcheck -x -s bash openhabian-setup.sh && echo "shellcheck openhabian-setup.sh - OK"`

Reply:
ok